src/features/todo/todoSlice.js

```diff
@@ -0,0 +1,42 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+let todoId = 1
+
+const slice = createSlice({
+    name: 'todo',
+    initialState: [],
+    reducers : {
+        createTask: (state, action)=>{
+            state.push({
+                describtion: action.payload,
+                id: todoId,
+                isCompleted: false
+            })
+            todoId++
+        },
+        isCompleted: (state, action)=>{
```

Comment:
This implies that this function checks if a task is completed or not, however it does not. A more descriptive naming here would be `toggleCompleted` or `toggleIsCompleted`.